src/EnergyPlus/NodeInputManager.cc

```
@@ -176,16 +171,17 @@ void GetNodeNums(EnergyPlusData &state,
     }
 
     // Most calls to this routine use a fixed fluid stream number for all nodes, this is the default
-    FluidStreamNum = NodeFluidStream;
+    NodeInputManager::CompFluidStream FluidStreamNum = NodeFluidStream;
+    DataLoopNode::ConnectionType thisConnectionType;
     for (int Loop = 1; Loop <= NumNodes; ++Loop) {
 
         switch (nodeConnectionType) {
         case DataLoopNode::ConnectionType::Num:
         case DataLoopNode::ConnectionType::Invalid:
-            ConnectionType = ConnectionType::Invalid;
+            thisConnectionType = ConnectionType::Invalid;
             break;
         default:
-            ConnectionType = nodeConnectionType;
+            thisConnectionType = nodeConnectionType;
             break;
         }
```

Doesn't the connection type default to invalid? And isn't thisConnectionType always equal to nodeConnectionType (except for Num which should never happen)?

---

This must be leftover from the conversion from string to enum. It's gone now. And the optional_bool `IncrementFluidStream` is now a simple bool. Need to wait for CI to ensure nothing got broken here.